Symbology/qgis/RSContext/ownership.qml

```diff
@@ -0,0 +1,753 @@
+<!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
+<qgis maxScale="0" simplifyDrawingTol="1" simplifyAlgorithm="0" simplifyMaxScale="1" version="3.18.2-Zürich" hasScaleBasedVisibilityFlag="0" simplifyLocal="1" labelsEnabled="0" simplifyDrawingHints="1" styleCategories="AllStyleCategories" minScale="100000000" readOnly="0">
+  <flags>
+    <Identifiable>1</Identifiable>
+    <Removable>1</Removable>
+    <Searchable>1</Searchable>
+    <Private>0</Private>
+  </flags>
+  <temporal endField="" enabled="0" mode="0" durationUnit="min" durationField="" accumulate="0" fixedDuration="0" endExpression="" startExpression="" startField="">
+    <fixedRange>
+      <start></start>
+      <end></end>
+    </fixedRange>
+  </temporal>
+  <renderer-v2 enableorderby="0" type="categorizedSymbol" symbollevels="0" forceraster="0" attr="ADMIN_AGEN">
+    <categories>
+      <category label="Fish and Wildlife Service" value="FWS" symbol="0" render="true"/>
+      <category label="Other" value="UND" symbol="1" render="true"/>
+      <category label="US Army Corps of Engineers" value="USACE" symbol="2" render="true"/>
+      <category label="US Forest Service" value="USFS" symbol="3" render="true"/>
+      <category label="Private" value="PVT" symbol="4" render="true"/>
+      <category label="National Park Service" value="NPS" symbol="5" render="true"/>
+      <category label="Bureau of Land Management" value="BLM" symbol="6" render="true"/>
+      <category label="Bureau of Indian Affairs" value="BIA" symbol="7" render="true"/>
+      <category label="State" value="ST" symbol="8" render="true"/>
```

Comment:
@lauren-herbine can you control the display order in the legend of these? If so, is it simply by changing this `symbol` key? Perhaps we should make the order in the legend a little clearer?